dev/ci/ci-bedrock2.sh

```diff
@@ -5,7 +5,7 @@ ci_dir="$(dirname "$0")"
 
 bedrock2_CI_DIR="${CI_BUILD_DIR}/bedrock2"
 
-git_checkout "${bedrock2_CI_BRANCH}" "${bedrock2_CI_GITURL}" "${bedrock2_CI_DIR}"
+git_checkout "${bedrock2_CI_REF}" "${bedrock2_CI_GITURL}" "${bedrock2_CI_DIR}"
```

Comment:
Ideally we would just say `git_checkout bedrock` , but I lack the require sh-fu to make that happen.

Reply:
I can make this happen in a subsequent PR.